defaults/main.yml

```diff
@@ -146,10 +184,10 @@ neofs_ir__config: []
 neofs_ir__group_config: []
 neofs_ir__host_config: []
 
-neofs_ir__combined_config: '{{ 
+neofs_ir__combined_config: '{{
```

Comment:
All whitespace changes belong to a single commit.

Reply:
done